bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/EdCompTargetList.java

```diff
@@ -245,9 +245,14 @@ private void updateUIForGroup() {
     /**
      * Update the menu items that allow the addition of guide stars. These must be disabled for the automatic group
      * or if we are currently on the base or a user target.
+     *
+     * REL-2715 similarly demands that if there is only the auto guide group that they be enabled, with a new guide
+     * group created automatically.
      */
     private void updateGuideStarAdders() {
-        final boolean enabled = selectionIsManualGroup() || selectionIsManualTarget();
+        // If there is only one group, then it must be the auto group.
+        final boolean onlyAutoGroup = getContextTargetEnv().getGroups().size() == 1;
```

Comment:
This is certainly true but a bit indirect maybe.  We already have a `getAutomaticGroup` in `GuideEnvironment`, perhaps we should add a `getManualGroups`.